src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -159,6 +159,11 @@ public function firstOrFail($columns = array('*'))
 	 */
 	public function get($columns = array('*'))
 	{
+		if(is_string($columns))
```

Comment:
If you mean the space, the code style in Laravel is to have it like this: `if ($whatever)` and not `if($whatever)`.

If you mean squash to one commit, that basically means making your 5 commits into 1, that way if Taylor merges this it's not 5 commits in the commit history, it's only 1.  This makes it more clear, because rather than having 1 commit for the actual thing that the PR changed and 4 for code style fixes etc. it's 1 commit for just the "finished" PR if it's merged.  Some Googling should help you out with how to squash commits.